src/helpers.ts

```diff
@@ -1065,9 +1069,16 @@ Note that in this case, the contract deployment will not behave the same if depl
       } else {
         proxyAdminName = viaAdminContract.name;
         if (!viaAdminContract.artifact) {
-          proxyAdminDeployed = await partialExtension.get(proxyAdminName);
+          try {
+            proxyAdminDeployed = await partialExtension.get(proxyAdminName);
+          } catch (e) {
+            console.log(
```

Comment:
it should not be a `console.log` as this would show up everywhere including test output

Reply:
Ok I removed the log completely as I see you have done in here:

![Screenshot 2021-07-12 at 13 51 51](https://user-images.githubusercontent.com/37438950/125290548-56358380-e318-11eb-93ca-6e85416853fb.png)

Reply:
@wighawag just pinging.

Reply:
Hi Dylan, sorry did not find the time yet to fully review it, but in principle it looks good. Hopefully I ll find some time before I am back from my travel.